examples/blender/blender.py

```diff
@@ -50,12 +59,26 @@ async def worker(ctx: WorkContext, tasks):
     # worst-case time overhead for initialization, e.g. negotiation, file transfer etc.
     init_overhead: timedelta = timedelta(minutes=3)
 
+    # Set up logging.
```

Comment:
this seems like a piece of docstring that should _not_ be part of the example script - but rather it should be part of the documentation for `yapapi.log` module itself ...

Reply:
Moved to the docstring for `yapapi.log` in https://github.com/golemfactory/yapapi/pull/41/commits/ae4461c0f5a11e23a11d36c249eef9d0a42b7969